src/app/page.tsx

```diff
@@ -165,11 +155,11 @@ export default function Home() {
         width="500px"
         height="700px"
         border="1px solid"
-        borderColor="primary.main"
+        borderColor="#B8860B"
         p={2}
         spacing={3}
         mt={2}
-        bgcolor="background.paper"
+        bgcolor="#333333"
```

Comment:
set the colors in the Material UI theme in the ThemeContextWrapper instead of hardcoding them as hexcodes